apps/tangle-dapp/utils/splitTokenValueAndSymbol.ts

```diff
@@ -0,0 +1,28 @@
+type SplitNumericAndAlphabetsReturnType = {
+  value: number;
+  symbol?: string;
+};
+
+/**
+ *
+ * This function splits a string into a numeric (Token value) and alphabets (Token symbol) part.
+ * @param string - The string to split
+ * @returns - An object containing the numeric and alphabets part of the string
+ */
+export const splitTokenValueAndSymbol = (
+  string: string
+): SplitNumericAndAlphabetsReturnType => {
+  const regex = /^(\d+\.?\d*)\s*(\D+)$/;
```

Comment:
add comment here to explain the regex

Reply:
Let's use `/** */` for multiple lines comments. That also supports for the documenting the variable. 

![CleanShot 2023-11-22 at 18 10 40](https://github.com/webb-tools/webb-dapp/assets/60747384/e543324f-d262-40b1-b3e9-e1d6232fe231)